_layouts/home.html

```diff
@@ -1,70 +1,295 @@
+              "Diversity, Equity and Inclusion": 4
+            },
+            "Leadership & management": {
+              "Project management": 7,
+              "Planning and organising events (workshops, meet-ups, conferences etc)": 1,
+              "Grant/proposal writing": 4,
+              "Evaluation and assessment": 1,
+              "Agile project management methodologies": 5,
+              "Recruitment": 4,
+              "Resource management (time, budget)": 7,
+              "Strategy": 4,
+              "Change management": 3,
+              "Domain expertise": 3,
+              "Research methodologies": 1,
+              "Ethics": 5,
+              "Domain-specific technical skills (AI/ML, research data and analytics, visualisation, computational science)": 2,
+              "Legal and regulatory": 7,
+              "Developing domain-specific training": 2
+            },
+            "Communication & dissemination": {
+              "Delivering training": 1,
+              "Writing and publication": 4,
+              "Public science communication": 5,
+              "Peer science communication": 1,
+              "Content and knowledge management": 4
+            }
+          };
+
+          const DATA_B = {
+            "Technical skills (general)": {
+              "Collaborative software development": 3,
+              "Coding conventions": 2,
+              "Software licensing": 1,
+              "Programming": 4,
+              "Environmental impact awareness": 3
+            },
+            "Technical (specific)": {
+              "Visualization": 2,
+              "Parallelization": 4,
+              "User Experience": 3,
+              "Technical translation between researchers and RSEs": 1,
+              "Code optimization": 2
+            }
+          };
+
+          const DATA_C = {
+            "Interpersonal": {
+              "Collaboration & building connections": 4,
+              "People management": 4,
+              "Stakeholder management": 4,
+              "Diversity, Equity and Inclusion": 4
+            },
+            "Leadership & management": {
+              "Project management": 7,
+              "Recruitment": 4,
+              "Resource management (time, budget)": 7,
+              "Strategy": 4,
+              "Domain expertise": 3,
+              "Research methodologies": 1,
+              "Developing domain-specific training": 2
+            },
+            "Communication & dissemination": {
+              "Delivering training": 1,
+              "Content and knowledge management": 4
+            }
+          };
+
+          createSkillChart(DATA_A, chart_id = "chart-a", legend_id = "legend-a");
+          createSkillChart(DATA_B, chart_id = "chart-b", legend_id = "legend-b");
+          createSkillChart(DATA_C, chart_id = "chart-c", legend_id = "legend-c");
+        </script>
+
+      </div>
+    </div>
+  </section>
+  <!--<section class="pb-5 position-relative bg-gradient-dark mx-n3">
```

Comment:
Glad I looked through the source code - I think these sections will be a nice addition :)

Reply:
Yes, I think this layout is nice. I kept them in the comments so I can go back to them, but I have a copy. I'll remove them for now to avoid any confusion, and we can review again as the website/app takes shape.